oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/status/ServerStatusService.java

```diff
@@ -64,6 +64,10 @@ public void bootedNow(List<ApplicationConfiguration.ModuleConfiguration> configu
     }
 
     public void rebalancedCluster(long rebalancedTime) {
+        if (!bootingStatus.isBooted()) {
+            return;
+        }
```

Comment:
ServerStatusWatcher will be notified by the method `ServerStatusService#bootNow` right after bootstrap complete.

I think notice once is enough in bootstrap stage. If you concern about `cluster_rebalanced_time` telemetry metric, how about move this return judge after `cluster_rebalanced_time` creating?

Reply:
OK, thanks for the explanation. Then you need to move `bootingStatus.setBooted(true);` to the last line of the `bootedNow` method. Otherwise, the race condition could still happen, right?

> I think notice once is enough in bootstrap stage. If you concern about cluster_rebalanced_time telemetry metric, how about move this return judge after cluster_rebalanced_time creating?

That is not correct. Because actually, we don't do rebalancing notifications.